### ISSUE TITLE
Weird behaviour when running `pulldown marionette`

### ISSUE THREAD

**Issue by jackfranklin** [jackfranklin/pulldown#76]
```
public/javascripts/lib > pulldown marionette 
-> Success: https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.5.1/underscore-min.js was downloaded to ./underscore-min.js
-> Success: https://cdnjs.cloudflare.com/ajax/libs/backbone.js/1.0.0/backbone-min.js was downloaded to ./backbone-min.js
-> Success: https://cdnjs.cloudflare.com/ajax/libs/jquery/2.0.3/jquery.min.js was downloaded to ./jquery.min.js
-> Success: https://cdnjs.cloudflare.com/ajax/libs/backbone.marionette/1.1.0-bundled/backbone.marionette.min.js was downloaded to ./backbone.marionette.min.js
```

it grabs backbone set too?!

**Comment by tgvashworth** [comment]
Yep, I added it [as a set](http://pulldown-api.herokuapp.com/set/marionette).

**Comment by jackfranklin** [comment]
Ah nice, fair enough.

I wonder if to make this clearer, there should be an additional line of terminal output:

```
pulldown marionette
-> Pulldown found set "marionette": backbone, backbone.marionette
-> Success: .....
...
```

Perhaps?

**Comment by tgvashworth** [comment]
Hmm. That's a tricky one, because it all resolves itself inside pulldown-resolve (I think). Worth thinking about tho.

**Comment by jackfranklin** [comment]
Ah of course, yes. Hmm.

**Comment by jackfranklin** [comment]
@phuu it would be good to revisit this and have a think about if it's possible...

**Comment by tgvashworth** [comment]
The obvious way would be to hit the api endpoint for a particular set to see what you get back, but in the case of `marionette` it would be:

``` json
[ "backbone", "backbone.marionette" ]
```

Which ain't much use. But it's more useful than nothing, eh?

The next step would be to _partially_ resolve the identifier, util _just_ before it becomes a URL. But it's kinda the point of pulldown that it just deals in URLs – the resolving, API and set stuff is convenience.

**Comment by jackfranklin** [comment]
But the fact that the length > 1 tells you it's a set, right? So that would be enough to at least output "OMG SETTT"?

**Comment by tgvashworth** [comment]
Oh I just re-read what you said. Yes, we can defo do that. Just make the CLI hit the API. ALL THE ACRONYMS.

**Comment by jackfranklin** [comment]
Cool, I'll work on this sometime soon :)

**Comment by tgvashworth** [comment]
Result of chat elsewhere is to make pulldown-resolve and pulldown EventEmitters, then emit `resolve` events (or something) that the CLI can react to.

**Comment by jackfranklin** [comment]
+1. Let's get that nailed and then we can revisit this.
